packages/binding.template/spec/templatingBehaviors.ts

```diff
@@ -174,21 +177,21 @@ describe('Templating', function () {
     setTemplateEngine(new dummyTemplateEngine({ someTemplate: 'template output' }))
     testNode.innerHTML = "<div data-bind='template:\"someTemplate\"'></div>"
     applyBindings(null, testNode)
-    expect(testNode.childNodes[0].innerHTML).toEqual('template output')
+    expect((testNode.childNodes[0]as HTMLElement).innerHTML).toEqual('template output')
```

Comment:
small thing: whitespace before `as`

Reply:
oh, fixed in https://github.com/Auge19/tko/pull/4